src/data/types.ts

```diff
@@ -118,3 +118,12 @@ export type ClubSponsor = {
    */
   logoFilter?: string;
 }
+
+export type ClubSocials = {
+  instagram: string;
+  facebook: string;
+  medium: string;
+  github: string;
+
```

Comment:
Nit: whitespace or is that your code formatter?

Reply:
Intentional, since in my mind email felt a bit different from the others - but kind of pointless, so I'll remove the whitespace haha